app/jobs/sms_sender_otp_job.rb

```diff
@@ -16,9 +16,7 @@ def send_otp(twilio_service, code, mobile)
 
   def otp_message(code)
     <<-END.strip_heredoc
-      You requested a secure one-time password to log in to your #{APP_NAME} Account.
-
-      Please enter this secure one-time password: #{code}
+      #{code} is your #{APP_NAME} one-time password.
```

Comment:
Remove the << thing too?

I prefer: "Your one-time password fro Login.gov is 123355", not for security reasons, just because it reads better.